pkgs/desktops/gnome-3/misc/libgda/default.nix

```diff
@@ -11,16 +17,23 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  configureFlags = [
-    "--enable-gi-system-install=no"
+  NIX_CFLAGS_COMPILE = stdenv.lib.optionals mysqlSupport [
```

Comment:
In my packaging, MySQL works for me even without  `NIX_CFLAGS_COMPILE`. My lazy attempt without conditionals:

```patch
--- a/pkgs/desktops/gnome-3/misc/libgda/default.nix
+++ b/pkgs/desktops/gnome-3/misc/libgda/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, fetchpatch, pkgconfig, intltool, itstool, libxml2, gtk3, openssl }:
+{ stdenv, fetchurl, fetchpatch, pkgconfig, intltool, itstool, libxml2, gtk3, openssl, postgresql, mysql }:
 
 stdenv.mkDerivation rec {
   inherit (import ./src.nix fetchurl) name src;
@@ -20,7 +20,7 @@
   hardeningDisable = [ "format" ];
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ intltool itstool libxml2 gtk3 openssl ];
+  buildInputs = [ intltool itstool libxml2 gtk3 openssl postgresql mysql.connector-c ];
 
   meta = with stdenv.lib; {
     description = "Database access library";
```

Reply:
Nice, removed the cflags but it remained as optionals :+1: